src/conf_mode/interfaces_openvpn.py

```diff
@@ -257,13 +271,19 @@ def verify(openvpn):
         if openvpn['protocol'] == 'tcp-passive':
             raise ConfigError('Protocol "tcp-passive" is not valid in client mode')
 
+        if 'ip_version' in openvpn and openvpn['ip_version'] == 'ipv6-dual-stack':
+            raise ConfigError('IP Protocol "ipv6-dual-stack" is not supported in client mode')
```

Comment:
In what sense is it not supported? This config works just fine:

```
client
proto udp
remote 192.0.2.3
remote 2001:db8:1::1
```

But again, I think the dual-stack option should be just an implicit default.

Reply:
Discussed in the thread above.